pkg/rancher-components/package.json

```diff
@@ -32,8 +32,8 @@
     "@vue/test-utils": "1.2.1",
     "babel-eslint": "10.1.0",
     "core-js": "3.25.3",
-    "cron-validator": "1.2.0",
```

Comment:
I think we don't need to update the chron library for `rancher-components`, but It doesn't hurt.
The new lib version is something we have to test when doing the migration though.